### PR TITLE
Development subsquid

### PR DESCRIPTION
**Pull request by maayarosama**
### Description

Added a new weblet solution for subsquid indexer.
### Changes

Added A new solution
### Related Issues

#951

### CODE REVIEW

File: src/elements/subsquid/Subsquid.wc.svelte [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+<div style="padding: 15px;">
+  <form class="box" on:submit|preventDefault={deploySubsquidHandler}>
+    <h4 class="is-size-4 mb-4">Deploy a Subsquid Instance</h4>
+    <p>
+      Subsquid indexer is a piece of software that reads all the blocks from a
+      Substrate based blockchain, decodes and stores them for processing in a
+      later stage.
+      <a
+        target="_blank"
+        href="https://library.threefold.me/info/manual/#/manual__weblets_presearch"
```
Comment (DylanVerstraete) [inline src/elements/subsquid/Subsquid.wc.svelte:101]: another url maybe?

Comment (maayarosama) [reply src/elements/subsquid/Subsquid.wc.svelte:101]: As there's no current documentation for subsquid in the threefold manual, Maybe we should remove the reference link for now and added later on when the documentation is available?

Comment (AhmedHanafy725) [reply src/elements/subsquid/Subsquid.wc.svelte:101]: please remove the link for now and create 2 issues
- one on the manual https://github.com/threefoldfoundation/info_manual3/issues to add docs for the solution
- and the other here in weblets to add the link back once the docs are added

Comment (AhmedHanafy725) [reply src/elements/subsquid/Subsquid.wc.svelte:101]: please remove the link for now and create 2 issues
- one on the manual https://github.com/threefoldfoundation/info_manual3/issues to add docs for the solution
- and the other here in weblets to add the link back once the docs are added

File: src/utils/deploySubsquid.ts [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  vm.node_id = nodeId;
+  vm.disks = [disk];
+  vm.public_ip = publicIp;
+  vm.planetary = true;
+  vm.cpu = cpu;
+  vm.memory = memory;
+  vm.qsfs_disks = [];
+  vm.rootfs_size = rootFs(cpu, memory);
+  vm.flist =
+    "https://hub.grid.tf/omarabdulaziz.3bot/omarabdul3ziz-subsquid-22.08.flist";
```
[the remark this answers is not in the document]
Comment (maayarosama) [reply src/utils/deploySubsquid.ts]: Done

File: src/elements/subsquid/Subsquid.wc.svelte (hunk 1 above)
Comment (maayarosama) [reply src/elements/subsquid/Subsquid.wc.svelte:101]: Done

**Review by MohamedElmdary — APPROVED** [review]

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**